spec/lib/email_alert_signup_api_spec.rb

```diff
@@ -445,7 +445,7 @@ def init_simple_email_alert_api(subscription_url)
       it 'asks email-alert-api to find or create the subscriber list' do
         expect(Services.email_alert_api).to receive(:find_or_create_subscriber_list).with(
           "links" => {
-            "facet_groups" => { any: %w(52435175-82ed-4a04-adef-74c0199d0f46) },
+            facet_groups: { any: %w(52435175-82ed-4a04-adef-74c0199d0f46) },
```

Comment:
its because I removed the (facet_groups)[https://github.com/alphagov/finder-frontend/blob/cdc184481c56828e8e8b1e91ae1052f16397429d/lib/email_alert_signup_api.rb#L61] method which had a string as a key and instead it now uses the (tags)[https://github.com/alphagov/finder-frontend/blob/cdc184481c56828e8e8b1e91ae1052f16397429d/lib/email_alert_signup_api.rb#L64] method. It's a bit messy that there is a mixture of symbol and string keys but at least it's more consistently messy now!